layouts/_default/baseof.html

```diff
@@ -24,43 +24,16 @@
     {{ partial "footer/footer.html" . }}
     {{ partial "footer/script-footer.html" . }}
 
-    <!-- Google Tag Manager -->
-    <script type="text/plain" data-cookiecategory="analytics">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-TR443T9');</script>
-    <!-- End Google Tag Manager -->
-
-    <!-- Send Custom Event to Google Analytics for the live search functionality. -->
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-HCQWPQKQH8"></script>
     <script>
-      window.addEventListener('DOMContentLoaded', () => {
-        const searchSuggestionsWrapper = document.getElementById('suggestions');
-        if (!searchSuggestionsWrapper) return;
-
-        searchSuggestionsWrapper.addEventListener('click', (event) => {
-          const target = event.target;
-
-          for (const suggestionElement of searchSuggestionsWrapper.children) {
-            if (!suggestionElement.contains(target)) continue;
-
-            // This guards when the event is triggered on the padding area of the "search result" element.
-            // The page from the result will only be loaded when clicked on the <a> or any element inside.
-            if (target === suggestionElement) return;
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
 
-            const suggestionTitle = suggestionElement.querySelector('.suggestion__title');
-
-            // When there are no results for the search term.
-            if (!suggestionTitle) return;
-            const searchTerm = document.getElementById('search').value;
-
-            window.dataLayer.push({
-              'event': 'liveSearch',
-              'searchTerm': searchTerm
-            });
-          }
-        });
-      });
+      gtag('config', {{ .Site.Params.tagManagerId }});
```

Comment:
Do we need to set the `tagManagerId` for each documentation? Or maybe when we split the google analytics boards for each documentation? I am not sure what we discussed about that.

Reply:
Not really. It would be nice to separate the - the docs also suggest using different boards (or at least tag managers) for different domains, but currently we won't do it.